src/legacy/core_plugins/console/public/src/input.js

```diff
@@ -139,6 +139,7 @@ export function initializeInput($el, $actionsEl, $copyAsCurlEl, output, openDocu
 
             // we have someone on the other side. Add to history
             history.addToHistory(esPath, esMethod, esData);
+            addedToHistoryCb();
```

Comment:
On around line 215, a keyboard shortcut is added that lets the user hit Ctrl-Enter to execute whichever request their cursor is inside of. This means `sendCurrentRequestToES` is called when the user hits Ctrl-Enter. However, the keyboard-shortcut code passes in a non-function argument as the `addedToHistoryCb ` parameter, and this causes a silent failure when line 142 attempts to call it. This silent failure causes no results to show up in the response pane, despite the request successfully being executed.

We can fix this by first wrapping this in a condition:
```js
if (addedToHistoryCb) {
  addedToHistoryCb();
}
```

And then on line 215 we can change the keyboard shortcut to call`sendCurrentRequestToES` without arguments:

```js
  input.commands.addCommand({
    name: 'send to elasticsearch',
    bindKey: { win: 'Ctrl-Enter', mac: 'Command-Enter' },
    // Don't pass any arguments to sendCurrentRequestToES.
    exec: () => sendCurrentRequestToES()
  });
```